core-bundle/tests/Contao/StringUtilTest.php

```diff
@@ -68,30 +68,6 @@ public function parseSimpleTokensProvider(): \Generator
             'This is my test@foobar.com,foo@test.com',
         ];
 
-        yield 'Test token replacement with special characters (-)' => [
-            'This is my ##e-mail##',
-            ['e-mail' => 'test@foobar.com'],
-            'This is my test@foobar.com',
-        ];
-
-        yield 'Test token replacement with special characters (&)' => [
-            'This is my ##e&mail##',
-            ['e&mail' => 'test@foobar.com'],
-            'This is my test@foobar.com',
-        ];
-
-        yield 'Test token replacement with special characters (#)' => [
-            'This is my ##e#mail##',
-            ['e#mail' => 'test@foobar.com'],
-            'This is my test@foobar.com',
-        ];
-
-        yield 'Test token replacement with token delimiter (##)' => [
-            'This is my ##e##mail##',
-            ['e##mail' => 'test@foobar.com'],
-            'This is my ##e##mail##',
-        ];
-
```

Comment:
could these tests be reverted now that we restored the original functionality?

Reply:
they are still there...

Reply:
@aschempp see line 329 `parseSimpleTokensLegacyProvider()`